Automattic/wp-calypso
# Composite checkout: update button styles

#### Changes proposed in this Pull Request

I was looking at our DevDocs and realized I took a little too much liberty with the [button](https://wpcalypso.wordpress.com/devdocs/design/button) and [radio styles](https://wpcalypso.wordpress.com/devdocs/design/form-fields). In this PR, I have updated the buttons and radios to match the Calypso styles closer. 

**Before**
![image](https://user-images.githubusercontent.com/6981253/69366376-3f827a80-0c64-11ea-971b-15715a7a45f6.png)


**After**
![image](https://user-images.githubusercontent.com/6981253/69366356-32fe2200-0c64-11ea-9d76-7adbfd79df58.png)


#### Testing instructions

* Get checkout [running locally](https://github.com/Automattic/wp-calypso/pull/37013).
* Roll over and click on buttons to make sure they match DevDocs.
